configure.ac

```diff
@@ -5142,6 +5142,145 @@ then
     AM_CFLAGS="$AM_CFLAGS -DWOLFSSL_DES_ECB -DWOLFSSL_SIGNER_DER_CERT"
 fi
 
+# wolfEngine Options
+AC_ARG_ENABLE([engine],
+    [AS_HELP_STRING([--enable-engine],[Enable wolfEngine options (default: disabled)])],
+    [ ENABLED_WOLFENGINE=$enableval ],
+    [ ENABLED_WOLFENGINE=no ]
+    )
+
+AS_IF([test "x$ENABLED_WOLFENGINE" = "xyes"],[AM_CPPFLAGS="$AM_CPPFLAGS -DWOLFSSL_WOLFENGINE"])
```

Comment:
Are `WOLFSSL_WOLFENGINE` and `WOLFSSL_WOLFFIPSENGINE` used anywhere? If not, I don't think they need to be added.

Reply:
Nope.  I was just following what other flags were doing.  I will remove that tomorrow.